BussinesObject/Ui/UiEntities/TestRunsRecord.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Core.Selenium.Attributes;
+using Core.Selenium.WebElements;
+using OpenQA.Selenium;
+
+namespace BussinesObject.Ui.UiEntities;
+
+#pragma warning disable CS8618
+
+public class TestRunsRecord : RecordBase
+{
+    [RecordIndex(0)]
+    [RecordLocator("./child::div")]
+    public string Name { get; set; }
+
+    [RecordIndex(1)]
+    [RecordLocator(".//div[@class='period']")]
+    public string Period { get; set; }
+
+    private ActionTestRun action;
+    public ActionTestRun Action
+    {
+        get
+        {
+            return action;
+        }
+        set
+        {
+            action = value;
+            switch (action)
+            {
+                case ActionTestRun.Run:
+                    AddAction(new ActionRecord(4, By.XPath(".//a")));
+                    break;
+            }
+        }
+    }
+
+    public enum ActionTestRun
```

Comment:
separete class

Reply:
There is an Enum for each "Record" class, so I thought that it would be redundant to separate them, so it can only confuse